src/modules/data/board/Board.store.ts

```diff
@@ -292,13 +293,33 @@ export const useBoardStore = defineStore("boardStore", () => {
 		if (!board.value) return;
 
 		const {
+			cardId,
 			newIndex,
 			oldIndex,
 			forceNextTick,
 			fromColumnIndex,
 			toColumnIndex,
+			toColumnId,
 		} = payload;
 
+		let toColumn: ColumnResponse | undefined =
+			board.value.columns[toColumnIndex];
+		if (toColumn === undefined) {
+			toColumn = board.value.columns.find((column) => column.id === toColumnId);
+			if (toColumn === undefined) {
+				// cancel execution if target column does not exist
+				return;
+			}
+		}
```

Comment:
```suggestion
		const toColumn: ColumnResponse | undefined =
			board.value.columns[toColumnIndex] ??
			board.value.columns.find((column) => column.id === toColumnId);

		if (!toColumn) {
			// cancel execution if target column does not exist
			return;
		}
```
I think these lines above are more understandable, and reduce the nested if statements.

Reply:
removed